_blog-src/_posts/2015-07-07-fuse-apis.markdown

```diff
@@ -0,0 +1,171 @@
+---
+layout: post
+title:  "Manage Fuse APIs with apiman"
+date:   2015-07-07 15:15:15
+author: Stefan Bunciak
+categories: api management jboss fuse
+---
+
+This article aims to provide a short guide on how to get API Management capabilities provided by apiman to work with JBoss Fuse, a lightweight, flexible, integration platform that is based on [Apache Camel](http://camel.apache.org), an implementation of many of the most commonly used enterprise integration patterns (EIP).
+
+<!--more-->
+
+Creating API provider in JBoss Fuse
+====================================
+
+The following few steps should quickly get you started with deploying applications into JBoss Fuse. For demonstration purposes I'm using a sample REST application (quickstart) shipped along with Fuse installation. For more information about developing applications for JBoss Fuse inspect [developer materials](http://www.jboss.org/products/fuse/developer-materials/#!project=fuse) and other [resources](http://www.jboss.org/products/fuse/resources/) at jboss.org.
```

Comment:
- JBoss Fuse inspect [developer materials]
- JBoss Fuse inspect the [developer materials]